.dsp_cache

```diff
@@ -0,0 +1,3 @@
+info=<Dirac library cache file - do not edit>
+version=<3.6.2 [1305130900] LE>
+params=<df1108d090b4fc4a289a369fb9e946ce355eb7fc1782f1f59c41231c1e8a2cc18a9fa03731fc82302399c7b150dd85bfbe7529806ade111161b53399873070ce1b90764a78aab0f4e78b82b3acdc3bf28c2e75459072d28c16f10c3e3213980117d157da9643c5bc081ec4c2e9f684d1aad8a3823fe8ba7008a5768f39e8b2558073e6027dfbb9b6a22642636657a78398a98991f68945ec23050a6343cd20ae8715a2aaf269728e425a9dd9322f460a1224ec1a78f3bd297907b17f824e23a3bc4188bde50905f3fb2cd3686f6e1101e3905ecd9fc79d01cd39a208448690264b771fce9e432e1d32617f9e327d9ea9>
```

Comment:
I think this is a macOS metadata file that shouldn't be in the commit

Reply:
I thought i removed that from commit... my fault